### ISSUE TITLE
DLLVersion.rc is excluded from build, missing detail information in properties of *.dll

### ISSUE THREAD

**Issue by DomXC** [pocoproject/poco#2243]
#### Expected behavior

DLLVersion.rc is built for "debug_shared" and "release_shared" configuration to show detail information in properties of *.dll file like version, company, etc.

#### Actual behavior

Building of DLLVersion.rc in "debug_shared" and "release_shared" configuration is excluded from build, then there are missing detail information in properties of *.dll file like version, company, etc.

#### Steps to reproduce the problem

Open _vs140.sln / _vs150.sln -> build the solution

#### POCO version

1.8.0.1 and later

#### Compiler and version

VC140, VC150

#### Operating system and version

Windows 7 and later

#### Other relevant information

From my point of view, build of DLLVersion.rc should be enabled (Solution Explorer -> DLLVersion.rc -> Properties -> Exclude From Build -> NO

In other words, *_vs140.vcxproj / *_vs150.vcxproj should contain (x86 version):

```
    <ResourceCompile Include="..\DLLVersion.rc">
      <ExcludedFromBuild Condition="'$(Configuration)|$(Platform)'=='debug_static_md|Win32'">true</ExcludedFromBuild>
      <ExcludedFromBuild Condition="'$(Configuration)|$(Platform)'=='debug_static_mt|Win32'">true</ExcludedFromBuild>
      <ExcludedFromBuild Condition="'$(Configuration)|$(Platform)'=='release_static_md|Win32'">true</ExcludedFromBuild>
      <ExcludedFromBuild Condition="'$(Configuration)|$(Platform)'=='release_static_mt|Win32'">true</ExcludedFromBuild>
    </ResourceCompile>
```

instead of

```
    <ResourceCompile Include="..\DLLVersion.rc">
      <ExcludedFromBuild Condition="'$(Configuration)|$(Platform)'=='debug_shared|Win32'">true</ExcludedFromBuild>
      <ExcludedFromBuild Condition="'$(Configuration)|$(Platform)'=='debug_static_md|Win32'">true</ExcludedFromBuild>
      <ExcludedFromBuild Condition="'$(Configuration)|$(Platform)'=='debug_static_mt|Win32'">true</ExcludedFromBuild>
      <ExcludedFromBuild Condition="'$(Configuration)|$(Platform)'=='release_shared|Win32'">true</ExcludedFromBuild>
      <ExcludedFromBuild Condition="'$(Configuration)|$(Platform)'=='release_static_md|Win32'">true</ExcludedFromBuild>
      <ExcludedFromBuild Condition="'$(Configuration)|$(Platform)'=='release_static_mt|Win32'">true</ExcludedFromBuild>
    </ResourceCompile>
```

The same for x64.

**Comment by DomXC** [comment]
Question: @aleks-f is the exclusion of DLLVersion.rc from build an intent? Thank you.

**Comment by obiltschnig** [comment]
It seems this issues has sneaked in with the 1.8.0 release. Not sure why, but will be fixed with next release.

**Comment by OgreTransporter** [comment]
I've added the file to CMake script in pr #2720.

**Comment by obiltschnig** [comment]
fixed in 1.9.1
